layout-webapp/src/main/webapp/vue-app/common-layout-components/components/manage-permissions/ManagePermissionItem.vue

```diff
@@ -22,7 +22,7 @@
       size="19"
       class="fas fa-users me-3" />
     <span
-      class="d-flex text-truncate font-weight-bold text-start text-color body-2 mt-2">
+      class="d-flex text-truncate font-weight-bold text-start text-color text-body mt-2">
```

Comment:
No need to specify the class here IMO since the default is `text-body` style

```suggestion
      class="d-flex text-truncate font-weight-bold text-start mt-2">
```